File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        longest = 0
        st = set(nums)

        for n in st:
            if (n - 1) not in st:
                length = 1
                while (n + length) in st:
                    length += 1
                longest = max(longest, length)
            
        return longest

File: Data Structures & Algorithms/house-robber/submission-1.cpp
class Solution {
public:
    int rob(vector<int>& nums) {
        vector<int> dp = nums;
        dp.resize(nums.size()+1, 0);
        for(int i = 2; i < nums.size(); i++){
            dp[i] = dp[i-2] + dp[i];
        }

        return max(dp[dp.size()-1], dp[dp.size()-2]);
    }
};
